src/components/applied-product-filters/applied-product-filters.tsx

```diff
@@ -11,6 +11,7 @@ interface AppliedProductFiltersProps {
     onClearFilters: (filters: ProductFilter[]) => void;
     onClearAllFilters: () => void;
     currency: string;
+    priceBounds: { lowest: number; highest: number };
```

Comment:
The component receives both `appliedFilters: { minPrice, maxPrice }` and `priceBounds: { lowest; highest }`. I don't understand the difference between them.

Reply:
`appliedFilters` are filters from URL. I use lowest/highest from `priceBounds` as default values if one of them is missing. Is it more clear if I rename `priceBounds` to `defaultPriceFilters`?

Reply:
Now I'm spoilt by the knowledge and it's difficult to judge, but maybe:
```ts
// Min and max prices in filtered results. Used to replace missing
// bounds ("$5 - ?" or "? - $25") when only one filter bound is set.
minPriceFound: number;
maxPriceFound: number;
```

I think this would have an issue when no products are found for the given filters?

Reply:
> I think this would have an issue when no products are found for the given filters?

No (if I get the question correctly). Price bounds don't depend on what filters are currently applied.

Reply:
But they do, screenshots from this branch, in both cases the price filter is `minPrice=10`:

![Screenshot 2024-10-09 at 19 44 46](https://github.com/user-attachments/assets/b7f6dfce-19c3-46cd-9f64-d7cde8ceb18e)
![Screenshot 2024-10-09 at 19 45 00](https://github.com/user-attachments/assets/56bc303a-ea8a-423d-9b18-5c0ece76ba0e)

Reply:
Ah, now it seems to me that the first version was good enough. What if I return as it was and add a comment that you suggested?
```tsx
/**
 * The highest and lowest product prices in the current category.
 * Used to replace missing bounds ("$5 - ?" or "? - $25") when only one filter bound is set.
 */
priceBounds: { lowest: number; highest: number };
```

Reply:
> But they do, screenshots from this branch, in both cases the price filter is minPrice=10:

I am confused. Those are different categories on screenshots.

Reply:
@alisey I've pushed new chages.